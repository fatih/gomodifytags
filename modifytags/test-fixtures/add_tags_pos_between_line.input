package foo

type foo struct {
	bar string
	t   bool
	qux string
	yoo string
}
