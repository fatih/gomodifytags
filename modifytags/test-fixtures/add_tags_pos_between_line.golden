package foo

type foo struct {
	bar string
	t   bool   `json:"t,omitempty"`
	qux string `json:"qux,omitempty"`
	yoo string `json:"yoo,omitempty"`
}
